tools/Nethermind.Tools.Kute/Auth/JwtAuth.cs

```diff
@@ -32,12 +31,12 @@ public JwtAuth(ISystemClock clock, ISecretProvider secretProvider)
 
     private string GenerateAuthToken()
     {
-        var signingKey = new SymmetricSecurityKey(_secret);
-        var credentials = new SigningCredentials(signingKey, SecurityAlgorithms.HmacSha256);
-        var claims = new[] { new Claim(JwtRegisteredClaimNames.Iat, _clock.UtcNow.ToUnixTimeSeconds().ToString(), ClaimValueTypes.Integer64) };
-        var token = new JwtSecurityToken(claims: claims, signingCredentials: credentials);
-        var handler = new JwtSecurityTokenHandler();
+        var handler = new JsonWebTokenHandler { SetDefaultTimesOnTokenCreation = false };
 
-        return handler.WriteToken(token);
+        return handler.CreateToken(new SecurityTokenDescriptor
+        {
+            IssuedAt = _clock.UtcNow.UtcDateTime,
+            SigningCredentials = new(new SymmetricSecurityKey(_secret), SecurityAlgorithms.HmacSha256)
```

Comment:
Should we cache `SymmetricSecurityKey` instance?

Reply:
I tried not to make any other changes but replace the API as I don't know much about Kute. But the suggested change is small enough, so applied in the commit below.

Reply:
There should be no need for caching since we're already caching the JWT generation. By default, this code will be executed once per minute.